src/LanguageServer.spec.ts

```diff
@@ -129,19 +137,19 @@ describe('LanguageServer', () => {
         fsExtra.writeFileSync(srcPath, contents);
     }
 
-    describe('createStandaloneFileWorkspace', () => {
+    describe('createStandaloneFileProject', () => {
         it('never returns undefined', async () => {
             let filePath = `${rootDir}/.tmp/main.brs`;
             writeToFs(filePath, `sub main(): return: end sub`);
-            let firstWorkspace = await server['createStandaloneFileWorkspace'](filePath);
-            let secondWorkspace = await server['createStandaloneFileWorkspace'](filePath);
+            let firstWorkspace = await server['createStandaloneFileProject'](filePath);
```

Comment:
Should you update workspace styled variables to project?  Such as now `firstProject`, etc?

Reply:
Resolved